{
  "PRed": [
    {
      "PR": {
        "number": 1,
        "state": "closed"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "Version",
        "migrator_version": 0,
        "version": "1.8.7"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_version",
        "version"
      ]
    },
    {
      "PR": {
        "__lazy_json__": "pr_json/833050077.json"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "Version",
        "migrator_version": 0,
        "version": "1.8.8"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_version",
        "version"
      ]
    },
    {
      "PR": {
        "__lazy_json__": "pr_json/1001485795.json"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "Version",
        "migrator_version": 0,
        "version": "1.8.9"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_version",
        "version"
      ]
    },
    {
      "PR": {
        "__lazy_json__": "pr_json/1843100259.json"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "Version",
        "migrator_version": 0,
        "version": "1.8.10"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_version",
        "version"
      ]
    },
    {
      "PR": {
        "__lazy_json__": "pr_json/1957209503.json"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "MigrationYaml",
        "migrator_object_version": 1,
        "migrator_version": 0,
        "name": "python312"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_object_version",
        "migrator_version",
        "name"
      ]
    },
    {
      "PR": {
        "__lazy_json__": "pr_json/2034923484.json"
      },
      "data": {
        "bot_rerun": false,
        "migrator_name": "Version",
        "migrator_version": 0,
        "version": "1.8.11"
      },
      "keys": [
        "bot_rerun",
        "migrator_name",
        "migrator_version",
        "version"
      ]
    }
  ],
  "bad": false,
  "pinning_version": "2024.08.23.18.51.20",
  "pre_pr_migrator_attempt_ts": {
    "python310": 1758365596.0,
    "python311": 1756993996.0
  },
  "pre_pr_migrator_attempts": {
    "python310": 7,
    "python311": 388
  },
  "pre_pr_migrator_status": {
    "python310": "bot error (<a href=\"https://github.com/regro/autotick-bot/actions/runs/2844129436\">bot CI job</a>): main: Traceback (most recent call last):\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 1150, in _run_migrator\n    migrator_uid, pr_json = run(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 278, in run\n    eval_cmd(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/utils.py\", line 296, in eval_cmd\n    c.check_returncode()\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 460, in check_returncode\n    raise CalledProcessError(self.returncode, self.args, self.stdout,\nsubprocess.CalledProcessError: Command 'conda smithy rerender -c auto --no-check-uptodate' returned non-zero exit status 1.\n",
    "python311": "not solvable (<a href=\"https://github.com/regro/cf-scripts/actions/runs/6273073396\">bot CI job</a>): main: ['No `.ci_support/*.yaml` files found! This can happen when a rerender results in no builds for a recipe (e.g., a recipe is python 2.7 only). This attempted migration is being reported as not solvable.']"
  },
  "smithy_version": "3.39.0"
}